alibi_detect/utils/pytorch/distance.py

```diff
@@ -120,6 +120,13 @@ def permed_lsdds(
     Vector of B LSDD estimates for each permutation, H_lam_inv which may have been inferred, and optionally
     the unpermed LSDD estimate.
     """
+
+    # Check for overflow/underflow caused by too high dimensionality
+    if (~torch.isfinite(H)).any():
+        raise ValueError(
+            "Overflow or underflow occured. Trying reducing dimensionality or trying "
```

Comment:
small typo -> "try" instead of trying 2x

Reply:
Ah yeah thanks. Fixed.